CraftTweaker2-API/src/main/java/crafttweaker/api/liquid/WeightedLiquidStack.java

```diff
@@ -11,9 +12,9 @@
  *
  * @author Stan Hebben
  */
-@ZenClass("crafttweaker.item.WeightedLiquidStack")
+@ZenClass("crafttweaker.liquid.WeightedLiquidStack")
```

Comment:
While I understand this is a typo, we can't change it now without potentially breaking a bunch of scripts, so this needs to be reverted.